.gitlab/CODEOWNERS

```
@@ -415,6 +415,7 @@ Gemfile.lock                                   @baci/team_infrastructure
 .github/ISSUE_TEMPLATE/bug_report.md                                         @baci/baci_developers
 .github/ISSUE_TEMPLATE/enhancement.md                                        @baci/baci_developers
 .github/workflows/                                                           @baci/team_infrastructure
+.github/actions/                                                             @baci/team_infrastructure
```

```suggestion
.github/                                                             @baci/team_infrastructure
```
I think we could just name the `.github/` folder to save us the trouble to adapt in other PRs. The Gitlab codeowners file does not make sense anyway once we are fully on Github.

---

I left it for now like so as we also have the PR/Issue templates. We can change it later if we decide how we will proceed with codeowners.